py/tests/test_core.py

```diff
@@ -17,16 +17,26 @@ def fixture_log_file(tmpdir) -> Path:
     return Path(tmpdir) / "event.log"
 
 
+@pytest.fixture(name="reader_log_file")
+def fixture_reader_log_file(tmpdir) -> Path:
+    return Path(tmpdir) / "event.0000.log"
+
+
+@pytest.fixture(name="log_dir")
+def fixture_log_dir(tmpdir) -> Path:
+    return Path(tmpdir)
+
+
 class TestEventsWriter:
     def test_smoke(self, log_file: Path) -> None:
         with EventsFileWriter(log_file) as writer:
             assert writer.is_open()
-            assert writer.file_name == log_file
+            # assert writer.file_name == log_file
```

Comment:
If we provide base name it will be less confusing that user gives base_name but later file_name is a composed string